test/bpf/verifier-test.sh

```diff
@@ -138,25 +132,12 @@ function cg_prog_type_init {
 			attach_types["rcv-sock6"]="recvmsg6"
 		fi
 	fi
-
-	# Hack: Get the eppolicymap and sockmap pinned
-	#
-	# Basically none of the loaders support arbitrarily creating a map
-	# of type 'hash_of_maps' right now, so we have a little tool in the
-	# repo that allows creation of this map type from the commandline.
-	#
-	# Only set it up if we determined kernel support above!
-	if [ "${#attach_types[@]}" -gt 1 ]; then
-		$MAPTOOL eppolicymap "test_cilium_ep_to_policy" 2>/dev/null
-		$MAPTOOL sockmap "test_sock_ops_map" 2>/dev/null
-	fi
 }
 
 
 function load_cg {
 	cg_prog_type_init
 
-	mkdir -p $TESTMOUNT/sockops
```

Comment:
Sorry I think I made a mistake when asking you to remove this line. What we want is to avoid creating the `sockops` directory, but `mkdir -p` also creates the `$TESTMOUNT` directory if it does not exist and we shouldn't remove that. So I think this line should be instead:

```
	mkdir -p ${TESTMOUNT}
```

(Just like in `cg_prog_type_init()`.)